docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+services:
+  mongodb:
+    image: mongo
+    container_name: mongodb
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=${MONGO_INITDB_ROOT_USERNAME}
+      - MONGO_INITDB_ROOT_PASSWORD=${MONGO_INITDB_ROOT_PASSWORD}
+    ports:
+      - 27017:27017
+    volumes:
+      - ./mongo:/data/db
+    restart: unless-stopped
+  backend:
```

Comment:
It would probably be better practice to create a `Dockerfile` for the container since we don't need any volumes for that. For example, changing a file in the directory would affect the Docker container.